### PR TITLE
API-101 # Get AWS Credentials from Server CLI Service

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: lib/apis.js [hunk 1]
```diff
@@ -32,22 +32,11 @@ function getRouteConfig (
     .then((routeConfigs) => handlers.findRouteConfig(route, routeConfigs))
     .then((routeConfig) => {
       routeConfig.module = path.resolve(cwd, routeConfig.module)
-      wipeRouteFromRequireCache(routeConfig)
       return routeConfig
     })
 }
 
-function wipeRouteFromRequireCache (
```
Comment (k7n4n5t3w4rt) [inline lib/apis.js:40]: I don't know what this is but it seems like a good thing to not have.

File: lib/assume-aws-roles.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      })
+  })
+    .then((body) => ({
+      accessKeyId: body.Credentials.AccessKeyId,
+      secretAccessKey: body.Credentials.SecretAccessKey,
+      sessionToken: body.Credentials.SessionToken
+    }))
+}
+
+function assumeAWSRoleToDeploy (
```
Comment (k7n4n5t3w4rt) [inline lib/assume-aws-roles.js:51]: I kind of like this but, for my own benefit, I'd like to understand the rationale behind it.

File: lib/logs.js [hunk 3]
```diff
@@ -10,22 +10,20 @@ import type {
   BlinkMRCServer
 } from '../types.js'
 */
-
+// "arn:aws:logs:ap-southeast-2:069883150555:log-group:/aws/lambda/bm-matt-api-blinkm-io-dev"
```
Comment (k7n4n5t3w4rt) [inline lib/logs.js]: Should this be here still?

Comment (mymattcarroll) [reply lib/logs.js]: removed

**Pull request merged**